src/api/undeploy/helpers/schema/undeploy-service-validation.js

```diff
@@ -0,0 +1,14 @@
+import Joi from 'joi'
+
+import { environments } from '~/src/config/index.js'
+
+function undeployServiceValidation() {
```

Comment:
generally if this is an object then just export the object. If it needs values export a function

Reply:
could do. I just copied the deploy validation and trimmed it

Reply:
yeah its not a showstopper, more preference